dsgrid/config/annual_time_dimension_config.py

```diff
@@ -49,7 +51,20 @@ def check_dataset_time_consistency(self, load_data_df):
                 f"load_data {time_col}s do not match expected times. mismatch={mismatch}"
             )
 
-    def convert_dataframe(self, df, project_time_dim):
+    def get_time_dataframe(self):
+        time_col = self.get_timestamp_load_data_columns()
+        assert len(time_col) == 1, time_col
+        time_col = time_col[0]
+        schema = StructType([StructField(time_col, IntegerType(), False)])
+
+        model_time = self.list_expected_dataset_timestamps()
+        df_time = _get_spark_session.createDataFrame(model_time, schema=schema)
+        return df_time
+
+    def get_time_dataframe_in_model_timezone(self):
+        return self.get_time_dataframe()
+
+    def convert_dataframe(self, df=None, project_time_dim=None, df_meta=None):
```

Comment:
Can you add df_meta to the abstractmethod in the base class and add a docstring to explain what it is?